locale/fi/LC_MESSAGES/django.po

```diff
@@ -1570,6 +1573,13 @@ msgstr "Muokkaa resurssia"
 msgid "Create new resource"
 msgstr "Luo uusi resurssi"
 
+#, python-brace-format
+msgid "{source_resource.name} copied successfully."
+msgstr ""
+
+msgid "Something went wrong! Please contact admin."
+msgstr ""
```

Comment:
"Jokin meni vikaan! Ota yhteyttä ylläpitäjään."

Reply:
Added the missing translations.